TWLight/applications/management/commands/applications_example_data.py

```diff
@@ -208,6 +208,18 @@ def handle(self, *args, **options):
         for app_to_renew in random.sample(list(all_apps), num_to_renew):
             app_to_renew.renew()
 
+        for application in Application.objects.filter(
+            status=Application.PENDING, parent__isnull=False
+        ):
+            parent_application = Application.objects.get(pk=application.parent)
```

Comment:
since you're doing a field lookup here, you need to feed it a field value rather than a whole object.
```suggestion
            parent_application = Application.objects.get(pk=application.parent.pk)
```